HTMLSerializer.js

```diff
@@ -94,8 +119,14 @@ var HTMLSerializer = class {
     } else {
       this.html.push(`<${tagName.toLowerCase()} `);
       this.processAttributes(element);
+      this.processPseudoElements(element);
       this.html.push('>');
 
+      if (tagName == 'HEAD') {
```

Comment:
I created and pushed a possible fix.  I don't know if it's the best way to do it.  Let me know if you have any suggestions.